fili-core/src/main/java/com/yahoo/bard/webservice/util/Utils.java

```diff
@@ -251,10 +227,70 @@ public static void canonicalize(JsonNode node, ObjectMapper mapper, boolean pres
     /**
      * Create metrics from instance descriptors and store in the metric dictionary.
      *
-     * @param metricDictionary  The dictionary to store metrics in
-     * @param metrics  The list of metric descriptors
+     * @param metricDictionary The dictionary to store metrics in
+     * @param metrics          The list of metric descriptors
      */
     public static void addToMetricDictionary(MetricDictionary metricDictionary, List<MetricInstance> metrics) {
         metrics.stream().map(MetricInstance::make).forEach(metricDictionary::add);
     }
+
+    /**
+     * Given a JsonObjectNode, order the ObjectNode entries on the key recursively and
+     * replace context blocks with empty nodes.
+     * <p>
+     * This method is recursive.
+     *
+     * @param node            The root of the tree of json nodes.
+     * @param mapper          The object mapper that creates and empty node.
+     * @param preserveContext Boolean indicating whether context should be omitted.
```

Comment:
The formatting from your IDEA is not the @param name(2 spaces) description of our contribution spec.  You probably need to tweak your autoformatter settings.